index.html

```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/microsoft_logo.png"
                             alt="jslovers homepage"
```

Comment:
Please update all alt tags. We can leave the alt tag empty as the text below this is already having the information

Reply:
ok